gatsby-node.js

```diff
@@ -146,6 +148,10 @@ exports.sourceNodes = async (
       store,
       cache
     });
+    
+    const nodeType = (row.separateNodeType) ?
```

Comment:
The only issue I can think of is that that isn't falsey when we are expecting it to be (in regards to the failing test). Could it be an issue with an unresolved promise?